### DISCUSSION TITLE
How to show Sub-totals with Customizable Summaries and Drag Grouping?

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6576] [Help]
For Customizable Summaries, they used to show Subtotals when the data is Grouped but no longer do so:
https://serenity.is/demo/AdvancedSamples/CustomizableSummaries

Subtotals would go where the red line is in the image below
![image](https://user-images.githubusercontent.com/57414714/201366995-5afbd755-c6c6-4483-bead-6e3f91ad34be.png)

**Comment by VictorTomaili** [comment]
#6577

**Reply by VictorTomaili** [reply]
Probably this issue is not your case. The issue is related to our pro package about draggable grouping.

**Reply by VictorTomaili** [reply]
Update your Serenity.Assets and Serenity.Scripts to latest version to get fix.
